apps/E2E/src/TabList/pages/TabListPageObject.ts

```diff
@@ -0,0 +1,77 @@
+import { BasePage, By } from '../../common/BasePage';
+import { Attribute, AttributeValue } from '../../common/consts';
+import {
+  TABLIST_TESTPAGE,
+  HOMEPAGE_TABLIST_BUTTON,
+  TABLIST_TEST_COMPONENT,
+  FIRST_TAB,
+  SECOND_TAB,
+  THIRD_TAB,
+  FOURTH_TAB,
+  FIFTH_TAB,
+  TABLIST_CALLBACK_TEXT,
+} from '../consts';
+
+type Tab = 'First' | 'Second' | 'Third' | 'Fourth' | 'Fifth';
+
+class TabListPageObject extends BasePage {
+  async resetListSelection(): Promise<void> {
+    return (await this.getTab('First')).click();
+  }
+
+  async isTabSelected(selector: Tab): Promise<boolean> {
+    return await (await this.getTab(selector)).isSelected();
+  }
+
+  async isTabFocused(selector: Tab): Promise<boolean> {
+    return await this.compareAttribute(this.getTab(selector), Attribute.IsFocused, AttributeValue.true);
```

Comment:
Missing `await` before this.getTab()

Reply:
`compareAttribute` takes a promise, so no need for an await here.